plugins/webp-uploads/hooks.php

```diff
@@ -666,12 +666,24 @@ function webp_uploads_update_featured_image( $html, $post_id, $attachment_id ) {
  */
 function webp_uploads_wepb_fallback() {
 	// Get mime type transforms for the site.
-	$transforms = webp_uploads_get_upload_image_mime_transforms();
+	$transforms   = webp_uploads_get_upload_image_mime_transforms();
+	$image_format = get_option( 'perflab_modern_image_format' );
```

Comment:
This `perflab_modern_image_format` setting isn't always registered, right? So the `$image_format` could be `false` here. Does that matter?